website/server/controllers/api-v3/user.js

```diff
@@ -1508,9 +1511,13 @@ api.markPmsRead = {
   url: '/user/mark-pms-read',
   async handler (req, res) {
```

Comment:
unfortunately this is a breaking change that would make existing calls to this route fail and we can't accept it under API v3

Reply:
It would have to be moved to API v4

Reply:
But then it's probably going to create issues with the mobile apps using v4, right @phillipthelen ?

Reply:
The best option might be to add, in api v4, a new route to selectively mark PMs as read while keeping this one and its original behavior. Also I'm not sure having a `count` parameter is very useful as at the end we mark an entire conversation as read anyway. Instead of passing it we could count the number of messages to read directly from the database

Reply:
We would also need tests for the new route